### ISSUE TITLE
Class SessionInterface missing

### ISSUE THREAD

**Issue by thomaspapiernik** [magespecialist/m2-MSP_Security_Suite_Common#2]
When I run di:compile :

[RuntimeException]                                                         
Class MSP\SecuritySuiteCommon\Api\SessionInterface does not exist

This class is not in vendor/msp !

I need it thru twofactorauth.

**Comment by stevecantlay** [comment]
It looks like it was removed from tag 2.0.5 - which is causing the issue as it is still included in MSP\SecuritySuiteCommon\Controller\Stop\Index

**Comment by diwipl** [comment]
+1 it is currently not possible to run setup:di:compile when 2FA extension is installed:
```
[RuntimeException]
Class MSP\\SecuritySuiteCommon\\Api\\SessionInterface does not exist in [MSP\\SecuritySuiteCommon\\Controller\\Stop\\Index\\Interceptor]
```

**Comment by mvalenziano** [comment]
a tmp solution is to add "msp/security-suite-common" : "2.0.4" to the require list in your main composer.json then run composer update

**Comment by diwipl** [comment]
It is not a solution because this package is dependency of https://github.com/magespecialist/m2-MSP_TwoFactorAuth which is the package I'm actually installing.

**Comment by phoenix128** [comment]
Thank you for reporting this and sorry for making you wait.
This has been fixed in 2.0.6
